app/views/billing_subscriptions/_new_modal.html.erb

```diff
@@ -1,35 +1,37 @@
 <%- # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later -%>
 <div class='modal skinny' id='newBillingSubscriptionModal'>
-	<%= render 'common/modal_header', title: "Movin' on up!" %>
+  <%= render 'common/modal_header', title: "Movin' on up!" %>
 
-	<div class='modal-body'>
-		<% if current_role?(:super_admin) && @nonprofit %>
-			<p><strong>Super admin mode</strong></p>
-			<p><%= "Selected nonprofit: #{@nonprofit.name} (id #{@nonprofit.id})" %></p>
-		<% end %>
+  <div class='modal-body'>
+    <% if current_role?(:super_admin) && @nonprofit %>
+    <p><strong>Super admin mode</strong></p>
+    <p><%= "Selected nonprofit: #{@nonprofit.name} (id #{@nonprofit.id})" %></p>
+    <% end %>
 
-		<form class='cardForm' parsley-validate>
-			<!--= on 'submit' create_billing_sub -->
+    <form autocomplete='off' class='cardForm' parsley-validate>
```

Comment:
The autocomplete shouldn't be turned off on this specific form. It's used for payment by the nonprofit.